tensorflow/compiler/jit/xla_device.h

```diff
@@ -188,6 +188,8 @@ class XlaDevice : public LocalDevice {
 
   Status RefreshStatus() override TF_LOCKS_EXCLUDED(mu_);
 
+  bool IsRemoteCallAllowed() const override { return true; }
```

Comment:
Is this override needed for XlaDevice? also for FakeDevice, XlaCompilationDevice, SingleThreadedCpuDevice?

Reply:
Hi zhangyujing Thanks for the comment. I refined the the list of overrides and will push an update shortly.